### PR TITLE
Add Items and remove items functionality

### PR DESCRIPTION
**Pull request by jchernandez87**
*Changes: 😃 

*Add button now adds an item to the list.
*Remove button now deletes an item from the list.

### CODE REVIEW

File: src/modules/addTask.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+const removeTask = (e) => {
+  const element = e.target;
+  if (element.classList[0] === 'del-btn') {
+    element.parentElement.parentElement.remove();
+  }
+  updateData();
+};
+
+const clearAll = () => {
```
Comment (smcommits) [inline src/modules/addTask.js:114]: - [ ] **The "Clear all complete" functionality works only on the last to-do item in the list. Please fix this**

Comment (smcommits) [inline src/modules/addTask.js:114]: - [ ] **Please make sure that the function for the "Clear all completed" button is implemented using the Array.filter() method.**

As a part of the requirements for this project, you have to use `Array.filter()` method to implement the functionality for "Clear all Completed".

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @jchernandez87,  

### You have done an excellent job. The project appears to be fantastic. It reflects great efforts and good understanding. 
### To make this project even better, there are some issues that you still need to work on. You are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


 ## Required Changes ♻️
_Please check the inline comments below._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
# Congratulations! 🎉 

## Hi @jchernandez87 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations once again! 🎉 



### Keep Rocking!👏👏👏

**Pull request merged**